docs/openapi/schemas.yaml

```diff
@@ -26,6 +26,7 @@ AuditType:
     - 'broken-backlinks'
     - 'canonical'
     - 'sitemap'
+    - 'url-inspect'
```

Comment:
This is very GSC API specific. For the audit name, we should opt for a more generic name, something along the lines of `URL indexability`.